src/js/components/__tests__/__snapshots__/components-test.js.snap

```diff
@@ -2081,6 +2081,39 @@ Object {
           },
           "round": "large",
           "separator": " : ",
+          "size": Object {
+            "large": Object {
+              "icon": Object {
+                "size": "30px",
+              },
+            },
+            "small": Object {
+              "icon": Object {
+                "size": "18px",
+              },
+              "pad": Object {
+                "horizontal": "small",
+                "vertical": "xxsmall",
+              },
+            },
+            "xlarge": Object {
+              "icon": Object {
+                "size": "36px",
+              },
+              "pad": Object {
+                "horizontal": "medium",
+                "vertical": "xsmall",
+              },
+            },
+            "xsmall": Object {
+              "icon": Object {
+                "size": "small",
+              },
+              "pad": Object {
+                "horizontal": "small",
```

Comment:
This horizontal pad looks disproportionally large than the horizontal pad on other sizes -- is that intentional?

Reply:
It does look a little big but xsmall is too small so it probably needs to be a pixel value (which I think we'd do anyway to be more like how button's padding is done.)